### ISSUE TITLE
Something seems wrong with the released HD_Xray_Pretrain_MAE pretrained Model

### ISSUE THREAD

**Issue by likakakaka** [Event-AHU/Medical_Image_Analysis#9]

**Comment by likakakaka** [comment]
![image](https://github.com/user-attachments/assets/e5d91678-3f15-45e4-8d6e-f4bd2c4af74c)
Sorry. I make a mistake. I have fixed my code and this is my reconstruction. It seems better now. Thanks. But I still wanna ask that the problem of "the extra 'head.mlp' and 'head.last_layer' parameters in the pretrained.pt but not show up in the MAE architecture built in Medical_Image_Analysis-main\HD_Xray_Pretrain_MAE\pretrain\models\mae.py."

**Comment by liyhc** [comment]
What's wrong?

**Comment by likakakaka** [comment]
Thanks for answering. I just tried the released MAE model to show the reconstruction results. Below is my results. I find that when I set the lower mast radio, the reconstruction results seem worse. That seems unresonable and maybe the model is overfitting? Note: I replaced the unmasked area with the original image when the image was displayed. 
![图片1](https://github.com/user-attachments/assets/b73c8e31-5b98-4988-8c55-d182de3788fa)

**Comment by liyhc** [comment]
In our approach, we employ a high mask ratio (95% in chest area while other lower but greater than 70%) during training to encourage the model to learn more robust and context-aware representations. When the mask ratio is lowered, the model has access to more information from the original image, which might initially seem beneficial. However, our model has been optimized to perform well under high mask ratios, where it needs to rely heavily on the context and learned representations to reconstruct the masked regions but it able to learn more generalized features. Therefore, it is possible that the model may not utilize the available information as effectively, leading to suboptimal reconstruction results.
